golang/day_trader/src/main.go

```diff
@@ -237,14 +250,15 @@ func (s *server) SetSellAmount(ctx context.Context, req *pb.Command) (*pb.Respon
 			return nil, err
 		}
 		sell.insertSell(ctx)
-		trigger := createSellTrigger(ctx, req.UserId, req.Symbol, sell.Id, req.Amount)
-		return &pb.Response{Message: trigger.toString()}, nil
+		createSellTrigger(ctx, req.UserId, req.Symbol, sell.Id, req.Amount)
+		return &pb.Response{UserId: user.Id, Message: "Successfully set sell amount"}, nil
 	}
 	err = trigger.updateCashAmount(ctx, req.Amount, user)
 	if err != nil {
 		logErrorEvent(ctx, err)
 	}
-	return &pb.Response{Message: trigger.toString()}, err
+	return &pb.Response{UserId: user.Id, Message: "Successfully set sell amount"}, err
```

Comment:
I don't actually care about this but you don't think it is useful to tell the user what information is in their trigger?

Reply:
Yeah, that is a possibility. I didn't have a clear idea of how I wanted to implement that on the frontend, so I started simple. That does make sense though, I will make that change.